node/Cargo.toml

```diff
@@ -111,4 +111,4 @@ polkadot-service = { git = "https://github.com/paritytech/polkadot", branch = "r
 polkadot-test-service = { git = "https://github.com/paritytech/polkadot", branch = "release-v0.9.20" }
 
 # Moonbeam Dependencies
-parachain-staking = { git = "https://github.com/OAK-Foundation/moonbeam", default-features = false, branch = "oak-polkadot-v0.9.20" }
+parachain-staking = { git = "https://github.com/OAK-Foundation/moonbeam", default-features = false, branch = "nikhil/unvested-issuance" }
```

Comment:
TODO: point parachain-staking at release branch once https://github.com/OAK-Foundation/moonbeam/pull/9 is merged